config.py

```diff
@@ -78,21 +79,23 @@ class BaseConfig(object):
 
     @staticmethod
     def build_db_uri(
-        db_type_arg=DB_TYPE,
-        db_user_arg=DB_USERNAME,
-        db_password_arg=DB_PASSWORD,
-        db_endpoint_arg=DB_ENDPOINT,
-        db_name_arg=DB_NAME,
+        db_type_arg=os.getenv("DB_TYPE"),
+        db_user_arg=os.getenv("DB_USERNAME"),
+        db_password_arg=os.getenv("DB_PASSWORD"),
+        db_endpoint_arg=os.getenv("DB_ENDPOINT"),
+        db_name_arg=os.getenv("DB_NAME"),
```

Comment:
I'm not sure I'm following, 🤔.

Reply:
> I'm not sure I'm following, 🤔.

This was a continuation from the above comment. On line 44 there is a variable called DB_NAME that has this value, then why not use that here instead of fetching the environment variable again.

Reply:
Fixing it now. Thanks for pointing that out 😊

Reply:
@epicadk , done. Can you please check again? Thanks

Reply:
Looks good 💯